tensorbay/cli/utility.py

```diff
@@ -22,26 +22,178 @@
 from ..client.gas import DatasetClientType
 from ..client.log import dump_request_and_response
 from ..client.requests import logger
-from ..exception import InternalServerError, TensorBayException
+from ..exception import InternalServerError, TensorBayException, UnauthorizedError
 from .tbrn import TBRN
 
 _Callable = TypeVar("_Callable", bound=Callable[..., None])
+INDENT = " " * 4
 
 
-class ContextInfo(NamedTuple):
+class ContextInfo:
```

Comment:
Should other related functions also be included? Like `read_config`, `form_profile_value`.